Steer/src/O2MCApplication.cxx

```diff
@@ -86,9 +86,7 @@ void O2MCApplicationBase::ConstructGeometry()
       mModIdToName[mod->GetModId()] = mod->GetName();
     }
   }
-
   FairMCApplication::ConstructGeometry();
-
   std::ofstream voltomodulefile("MCStepLoggerVolMap.dat");
```

Comment:
why is this file touched? Should probably not be part of this PR.

Reply:
True, I'll fix it.